libs/genai/pyproject.toml

```diff
@@ -46,6 +54,10 @@ codespell = "^2.2.0"
 optional = true
 
 [tool.poetry.group.test_integration.dependencies]
+numexpr = { version = "^2.8.8", python = ">=3.9,<4.0" }
+google-api-python-client = "^2.114.0"
+google-cloud-datastore = "^2.19.0"
+langchain = { git = "https://github.com/langchain-ai/langchain.git", subdirectory = "libs/langchain" }
```

Comment:
same here, why do we need this change?

Reply:
@lkuligin 
I updated PR. Remove unused libs.